.github/workflows/update.yaml

```diff
@@ -0,0 +1,184 @@
+name: Updates SDK based on api-definitions
+
+on: 
+  workflow_dispatch:
+    inputs:
+      trigger-reason:
+        description: |
+          The comment that will be posted to the PR if any changes are added.
+          Used to provide context about the source of the update.
+        required: false
+
+jobs:
+  build-api-definitions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout api-definitions repo
+        uses: actions/checkout@v2
+        with:
+          repository: 'Rebilly/api-definitions'
+
+      - name: Install api-definitions dependencies
+        run: npm install
+
+      - name: Build api-definitions
+        run: npm run build
+
+      - name: Upload built definitions
+        uses: actions/upload-artifact@v2
+        with:
+          name: definitions
+          path: dist
+          retention-days: 1
+          if-no-files-found: error
+
+  generate-sdk:
+    runs-on: ubuntu-latest
+    needs: build-api-definitions
+    strategy:
+      matrix:
+        api: ['core', 'user']
+    steps:
+      - name: Prepare variables
+        id: prep
+        run: |
+          OPENAPI_URI=""
+          SERVICE_NAME=""
+          MAPPING_SEGMENT="core"
```

Comment:
What is this for?

Reply:
This declares a segment of a custom path to resource mapping used under the generator hood.